site/dev/common.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+REMOTE="iceberg_docs"
+
+create_or_update_docs_remote () {
+  # check if remote exists before adding it
+  git config "remote.${REMOTE}.url" >/dev/null || 
+    git remote add "${REMOTE}" https://github.com/apache/iceberg.git
+
+  git fetch "${REMOTE}"
+}
+
+pull_remote () {
+  local BRANCH="$1"
+  assert_not_empty "${BRANCH}"
+
+  git pull "${REMOTE}" "${BRANCH}" 
+}
+
+push_remote () {
+  local BRANCH="$1"
+  assert_not_empty "${BRANCH}"
+
+  git push "${REMOTE}" "${BRANCH}" 
+}
+
+install_deps () {
+  pip -q install -r requirements.txt --upgrade
+}
+
+assert_not_empty () {
```

Comment:
not a huge deal but may help to pass through an error message so we know which arg is missing

Echo "Missing an argument: $2 "